src/installer/utils.py

```diff
@@ -24,6 +24,8 @@
     cast,
 )
 
+from installer.scripts import _build_shebang
```

Comment:
Not sure that I like importing and using a "private" function. I think a better solution would be to extract the part of the logic that is required for this use case into another function and having the `_build_shebang` use that. That should also allow `_build_shebang` to remain a part of the class, as it was before.